### ISSUE TITLE
[1.01.0019] project only opens every other time

### ISSUE THREAD

**Issue by zadmarbella** [blackberry/VSPlugin#285]
1. open fallingblocks project
2. repeat 1 if not the other time.

Current behaviour:
user is unable to open the project 

![unabletoopenproject](https://f.cloud.github.com/assets/1478549/964265/2c065422-0513-11e3-95e6-a6cb6a2fe717.png)

Expected:
project opens

**Comment by dbrgss** [comment]
Fixed 

v1.01.0022

**Comment by zadmarbella** [comment]
Verified Fixed with
v1.01.0022

**Comment by zadmarbella** [comment]
v1.01.0022
reopening - reproducible on win 8 vs2010

**Comment by zadmarbella** [comment]
v.1.01.0026 
note: seems to happen more frequently on vs2012

**Comment by guarnold** [comment]
There is a MSBuild bug in VS 2012. To avoid that, set the property "Linker -> General -> Output File" to "$(LocalDebuggerCommand)"
Also, you have to open your VS 2012 app in Administrator Mode once. That happens because VS has to set the properties, and it does that only in Administrator Mode. I have just to verify if we have to do that only once for VS 2012 or once for each app.

**Comment by guarnold** [comment]
Available in build version 1.01.0031

**Comment by zadmarbella** [comment]
reproducible on 1.01.0033 vs2012 win8 no error notification just fails to load - looks to be intermittent. unclear on how to reproduce.

**Comment by dbrgss** [comment]
This behavior happens for various reasons.   I think it might be good to close this issue (as it's not reproducible) and open up a new issue of it happening.. that we can work on.

**Comment by srinivas4912** [comment]
It is verified using VS2013/12 plugin from 2.0.0018 on win 8.1 and Win7 64 bit and found as fixed.
